src/modules/cpu/kernel/resize.hpp

```diff
@@ -0,0 +1,1109 @@
+#include "rppdefs.h"
+#include "cpu/rpp_cpu_simd.hpp"
+#include "cpu/rpp_cpu_common.hpp"
+
+RppStatus resize_u8_u8_host_tensor(Rpp8u *srcPtr,
+                                   RpptDescPtr srcDescPtr,
+                                   Rpp8u *dstPtr,
+                                   RpptDescPtr dstDescPtr,
+                                   RpptImagePatchPtr dstImgSize,
+                                   RpptROIPtr roiTensorPtrSrc,
+                                   RpptRoiType roiType,
+                                   RppLayoutParams srcLayoutParams)
+{
+    RpptROI roiDefault;
```

Comment:
L14-L20 could be replaced too like - https://github.com/r-abishek/rpp/blob/ar/opt_resize_bilinear_host/src/modules/cpu/kernel/brightness.hpp#L15